src/test/web/templates/compressed-managed-cookie.ftl

```diff
@@ -1,3 +1,3 @@
 [#ftl/]
 [#-- @ftlvariable name="cookie" type="io.fusionauth.http.Cookie" --]
-${cookie.value}
\ No newline at end of file
+${(cookie.value)!"(null)"}
```

Comment:
- `cookie.value!"default"` - `cookie` must be defined, but will default if `value` is `null`
- `(cookie.value)!"default` - will default if `cookie` _or_ `value` is `null`